testkit/result.go

```diff
@@ -130,3 +130,42 @@ func (res *Result) CheckAt(cols []int, expected [][]interface{}) {
 	need := fmt.Sprintf("%s", expected)
 	res.require.Equal(need, got, res.comment)
 }
+
+// CheckContains checks whether the result contains the expected string
```

Comment:
From the CI:
linting

  ✘  https://revive.run/r#exported  comment on exported method Result.CheckContain should be of the form "CheckContain ..."  
  testkit/result.go:134:1